public/myScript/orders.js

```diff
@@ -0,0 +1,158 @@
+
+
+var dynamicTable = (function() {
```

Comment:
The overall problem of this table, that while it has a generic name like `dynamic table`, which makes an assumption that this table for dynamic data, this table implementation has notion about orders statuses, styling and other things, which makes it more `orders table`, than `dynamic table`